CMakeLists.txt

```diff
@@ -104,12 +151,35 @@ target_link_libraries(trajectories
   PUBLIC
     Eigen3::Eigen
 )
+
 install(
   TARGETS
     trajectories
 )
 
+
 add_library(viam-ur)
+
+if (VIAM_UR_DISABLE_APPIMAGE)
```

Comment:
Here, in the non-appimage case, we generate a header which knows the relative path out of the `bin` directory and into the "data" directory. We will use this later to navigate from our executable to our assets.